REQUIREMENTS.md

```diff
@@ -88,6 +95,11 @@ To recover from `django` circa 1977.
 $ pip-compile -P django requirements.in
 ```
 
+**TLDR:**
+The `-P` flag signals to pip-compile that you want the latest possible
+package. Without that flag, pip-compile looks at what you compiled before and
+tries to reuse that version if it still satisfies the requirements.
```

Comment:
If its unversioned the latest possible is the latest? Not sure what I would change.